tests/TestCase/Core/FunctionsTest.php

```diff
@@ -42,4 +42,39 @@ public function testEnv()
         $this->assertEquals('0', env('ZERO'));
         $this->assertEquals('0', env('ZERO', '1'));
     }
+
+    /**
+     * Test error messages coming out when debug is on
```

Comment:
"Test error messages coming out when debug is on, manually setting the stack frame"